packages/perseus-core/src/utils/split-perseus-item.ts

```diff
@@ -0,0 +1,39 @@
+import _ from "underscore";
+
+import {getPublicWidgetOptionsFunction} from "../widgets/core-widget-registry";
+import {getUpgradedWidgetOptions} from "../widgets/upgrade";
+
+import type {PerseusRenderer} from "../data-schema";
+
+export default function splitPerseusItem(
```

Comment:
Should this function be called `splitPerseusRenderer`? Or perhaps `getRendererPublicData` for consistency with the `get*PublicWidgetOptions` functions?

```suggestion
export default function getRendererPublicData(
```

Reply:
I did `splitPerseusItem` to be consistent with `scorePerseusItem` and `validatePerseusItem`. If it's all the same to you, I think I'll keep it as-is. As an aside, I think both `Renderer` and `PublicData` are misnomers.

- `PerseusRenderer` isn't a renderer, it's an object that describes an article/exercise that's passed to a renderer. Most of Webapp calls it `item` or `itemData` which is a little better IMO but also a little more vague.
- Both the full data and `PublicOptions` are public data, they're just public at different times. I think `StrippedData` or `AnswerlessData` would be more appropriate, but they both have their own issues.

Reply:
I think `item` and `itemData` are usually a slightly higher-level structure that includes hints and some other rarely-used data. We actually have a type in Perseus called [`PerseusItem`](https://github.com/Khan/perseus/blob/e63f83d0d89fd5b8e7aee3ab7248bcb19ec9be8a/packages/perseus-core/src/data-schema.ts#L202). 

So I agree `PerseusRenderer` is a misnomer... I wonder what would be a better name that isn't "item" (as that name is taken).